jakarta.batch.tck.util/src/main/java/ee/jakarta/tck/batch/util/vehicle/ejb/EJBVehicleRemote.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership. Licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package ee.jakarta.tck.batch.util.vehicle.ejb;
+
+public interface EJBVehicleRemote {
+  public void runTest(Runnable test);
```

Comment:
Maybe nitpicking but is "Remote" a bad name choice..since Runnable isn't Serializable, so it's really only going to work as a local EJB?

Reply:
This whole concept can't work like this for a real remote EJB. We would have to pass the name of the test instead of a Runnable, and the remote EJB would have to execute it again using JUnit5. That's roughly how Jakarta EE TCK does it too. So I think it really isn't relevant how we name this :) If we want to improve this concept, we would have to do some non-trivial coding to either extend Arquillian or implement our own solution on top of it.

Reply:
Right.  I was wondering if you named it that because you were planning on doing that.  It's fine though.